protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -2813,9 +2820,10 @@ message EstimateFeeResponse {
 
 // Request to fetch the estimated MarginLevels if an order were to trade immediately
 message EstimateMarginRequest {
+    option deprecated = true;
     // Market ID for the order.
     string market_id = 1 [(google.api.field_behavior) = REQUIRED];
-    // Party ID of the order.
+    // Market ID for the order
```

Comment:
Why was this changed? Looks like it's referencing the party ID string, not Market ID (also needs to keep the full stop)

Reply:
Some accidental change, reverted now.